src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework.Example/ExampleLanguageServer.cs

```diff
@@ -21,29 +21,33 @@ protected override ILspServices ConstructLspServices()
     {
         var serviceCollection = new ServiceCollection();
 
-        _ = AddHandlers(serviceCollection)
+        var _ = AddHandlers(serviceCollection)
             .AddSingleton<ILspLogger>(_logger)
             .AddSingleton<IRequestContextFactory<ExampleRequestContext>, ExampleRequestContextFactory>()
             .AddSingleton<IInitializeManager<InitializeParams, InitializeResult>, CapabilitiesManager>()
-            .AddSingleton<ILifeCycleManager>(GetLifeCycleManager())
             .AddSingleton(this);
 
+        var lifeCycleManager = GetLifeCycleManager();
```

Comment:
lifecycle manager is no longer required for basic impls since the default code always runs